### PR TITLE
Fix for gitignore getting written every time we run a build

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Fixed issue related to gitignore

### CODE REVIEW

**Review by michielvandergeest — APPROVED** [review]

**Comment by michielvandergeest** [comment]
LGTM - let's await if this fix solves the issues of the reporter, before we do the merge.

**Comment by ricardo-a-alves-alb** [comment]
LGTM (original reporter).

**Pull request merged**
